cypress/support/fragments/bulk-edit/bulk-edit-search-pane.js

```diff
@@ -683,7 +683,17 @@ export default {
       cy.expect(Button('Download errors (CSV)').exists());
     }
   },
-
+  verifyActionsAfterErrors(errors = true) {
```

Comment:
this is a redundant method, there are several methods that check these separately, please use those